stl/inc/format

```diff
@@ -756,6 +774,19 @@ public:
     }
 };
 
+// Fetch the value of an argument associated with a dynamic
+// width or precision specifier. This will be called with either
+// _Width_checker or _Precision_checker as "_Handler".
+template <class _Handler, class _FormatArg>
+_NODISCARD constexpr int _Get_dynamic_specs(const _FormatArg _Arg) {
+    _STL_INTERNAL_STATIC_ASSERT(_Is_any_of_v<_Handler, _Width_checker, _Precision_checker>);
+    const unsigned long long _Val = _STD visit_format_arg(_Handler(), _Arg);
+    if (_Val > (numeric_limits<int>::max)()) {

```

Comment:
Shouldnt that give a signed / unsigned warning?

Also can we simply use auto instead of unsigned long long

Reply:
I used the exact type just as documentation, makes things easier to read.